### ISSUE TITLE
MIT 5K Data pre-processing

### ISSUE THREAD

**Issue by mv-lab** [google-research/maxim#20]
Dear authors,

you mention a random partition for train/test of the MIT 5K dataset, instead of the "popular" adopted test set splited by [DeepUPE](https://github.com/dvlab-research/DeepUPE). How do you generate the input RGBs from the provided DNGs? (i.e. using lightroom software or other's method preprocessing?)

can you provide the ids of the validation set, so we can compare fairly?

Thank you.

**Comment by vztu** [comment]
Hi @mv-lab, thanks for bringing up this issue. Yes, there exist different versions of MIT 5K. I'm not following this field, so we just downloaded the preprocessed dataset provided in [UEGAN](https://github.com/eezkni/UEGAN) for convenience. We use the first 4500 and last 500 images for training and testing, following [MIRNet](https://github.com/swz30/MIRNet).
